x-pack/plugins/cases/public/containers/use_get_case_files.test.tsx

```diff
@@ -22,6 +22,7 @@ const hookParams = {
   page: 1,
   perPage: 1,
   searchTerm: 'foobar',
+  searchByType: [],
```

Comment:
Could you please add a test that confirms `filesClient.list` is called with the `mimeType` param when `searchByType` is defined?